catalog-info.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: contentful
+  description: |
+    JavaScript library for Contentful's Delivery API (node & browser).
```

Comment:
Maybe it's worth updating the description and mentioning it's supporting Typescript?